readthedocs/projects/querysets.py

```diff
@@ -131,27 +131,31 @@ def prefetch_latest_build(self):
         """
         from readthedocs.builds.models import Build
 
-        # Prefetch the latest build for each project.
-        subquery_build_latest = Subquery(
-            Build.internal.filter(project=OuterRef("project_id"))
-            .order_by("-date")
-            .values_list("id", flat=True)[:1]
+        # Get most recent and recent successful builds
+        builds_latest = (
+            Build.internal.filter(project__in=self)
+            .values("project")
```

Comment:
Do we need this line here? I understand the `project` value is not used.

Reply:
It might not, but this is for grouping by project. The latest build day per project is what is needed here. If there is a different way to group this, we don't need the second query at all.